### ISSUE TITLE
Add a new "Behavior" : "New tab to the right"

### ISSUE THREAD

**Issue by Arkane-x6** [crittermike/shortkeys#615]
It seems there are nothing to do that, it will be really useful to me

**Comment by dannypernik** [comment]
Agreed this would be nice in Shortkeys. In the meantime, try this extension:
https://chromewebstore.google.com/detail/fommpldicogfgegfpcoibmhkbbdpodjg

**Comment by crittermike** [comment]
In Shortkeys v5, you can achieve 'new tab to the right' using the 'Run JavaScript' action. However, the standard `chrome.tabs.create()` API already opens new tabs next to the current tab by default in modern Chrome. The built-in 'Open new tab' action should place the tab adjacent to the current one. If you need precise positioning, you can use the 'Run JavaScript' action. Closing as a feature request.
